main/acle.md

```diff
@@ -4746,6 +4764,39 @@ stored to memory is modified by replacing the low 32 bits of
 `value.val[0]` with the contents of the `ACCDATA_EL1` system register.
 The returned value is the same as for `__arm_st64bv`.
 
+## Atomic store with PCDPHINT intrinsics
+
+These intrinsics provide an atomic store intrinsic, which will
+make use of the `STSHH` hint instruction immediately followed by the
+associated store instruction.
+This intrinsic is available when `__ARM_FEATURE_PCDPHINT` is defined.
+
+To access these intrinsics, `<arm_acle.h>` should be included.
+
+``` c
+  // Variants are also available for:
+  // _s8, _u16, _s16, _u32, _s32, _u64, _s64, _u128,
```

Comment:
We don't need actual prototypes for these other variants? Or that is handled in some other (magic) way?